.github/OWNERS

```diff
@@ -1,7 +1,8 @@
 approvers:
+  - kimwnasptd
+  - thesuperzapper
+emeritus_approvers:
   - DavidSpek
   - elikatsis
-  - kimwnasptd
   - StefanoFioravanzo
-  - thesuperzapper
-  - yanniszark
+  - yanniszark
```

Comment:
Please, consider updating this file so that we keep the current content.

Similar may be for the other OWNERS file in this PR for the notebook-controller component.

Reply:
Agree, the funny think is that in our current active [branch](https://github.com/opendatahub-io/kubeflow/blob/v1.7-branch/.github/OWNERS) we don't have our names. Do you think to make this change now?

Reply:
yeah, looks like we updated just [the one in the root folder](https://github.com/opendatahub-io/kubeflow/blob/v1.7-branch/OWNERS). I am okay if it's updated as a part of this PR but also as a followup.

Reply:
As @jstourac pointed out, we only updated the root folder and that is enough for us, as root OWNER file take precedence our any other files in repo. 
we can remove the other OWNERs files in a separate PR to keep it clean, however we should make the root OWNER files with our changes.

Reply:
Alright, the OWNERS file on [root](https://github.com/atheo89/kubeflow/blob/RHOAIENG-10796/OWNERS ) folder includes our team already. So, next step after this PR get merged is to remove the rest sub-owners files.